plane/source/docs/soaring.rst

```diff
@@ -146,14 +147,14 @@ Set up RC switch (Optional)
 
 .. note::
 
-   Available in firmware revisions 4.1 and later.
+   Available in firmware revisions 4.1 and later. 
 
 
 You can use a 3-position RC switch to control when the autopilot can use soaring. Set the parameter SOAR_ENABLE_CH to the corresponding channel number. The 3 positions have the following effect.
 
- - Below 1500us. Soaring is disabled (equivalent to setting SOAR_ENABLE = 0). Throttle will be used as normal.
+ - Below 1500us. Soaring is disabled (equivalent to setting SOAR_ENABLE = 0). Throttle will be used as normal. Switching to this from either of the positions below, will disable Soaring and maintain the current flight mode.
  
- - 1500us to 1700us. Soaring will have control over throttle The mode will not automatically change to LOITER based on detected rising air. However, when manually set to LOITER mode using RC controller or GCS, the autopilot will try to follow rising air currents. It will still restore the previous mode if the aircraft is not climbing, or if it drifts too far (see below).
+ - 1500us to 1700us. Soaring will have control over throttle. The mode will not automatically change to LOITER based on detected rising air. However, when manually set to LOITER mode using RC controller or GCS, the autopilot will try to follow rising air currents. It will still restore the previous mode if the aircraft is not climbing, or if it drifts too far (see below). Switching to this position from the one below,  while LOITERing in a thermal has no effect until the thermal is exited.
```

Comment:
Maybe say "Switching to this position from above 1700us" or "from HIGH position" rather than "the one below", as that could be interpreted and being from the PWM range below, rather than the position below in the wiki text.

Reply:
fixed